package.json

```
@@ -107,8 +107,8 @@
     "@eslint-community/eslint-utils": "^4.2.0",
     "@eslint-community/regexpp": "^4.12.1",
     "@eslint/config-array": "^0.21.0",
-    "@eslint/config-helpers": "^0.3.0",
-    "@eslint/core": "^0.14.0",
+    "@eslint/config-helpers": "^0.2.1",
```

```suggestion
    "@eslint/config-helpers": "^0.3.0",
```

Probably a rebase error.